integration-tests/src/test/resources/features/processor.feature

```diff
@@ -0,0 +1,116 @@
+    And the Manager metric 'manager_processor_status_change_total{status="DELETED",}' count is at least 1
+
+  Scenario: Add a Processor with no action
+    Given authenticate against Manager
+    
+    When create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    
+    Then add a Processor to the Bridge "mybridge" with body is failing with HTTP response code 400:
+    """
+    {
+      "name": "noActionProcessor"
+    }
+    """
+
+  Scenario: Processor payload is malformed
+    Given authenticate against Manager
+    
+    When create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+    
+    Then add a Processor to the Bridge "mybridge" with body is failing with HTTP response code 400:
+    """
+    {
+      "name": "processorInvalid"
+      "filters": []
+    }
+    """
+
+  Scenario: Cannot access the list of Processors without authentication
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+    When logout of Manager
+    
+    Then the list of Processor instances of the Bridge "mybridge" is failing with HTTP response code 401
+
+  Scenario: Cannot create a processor without authentication
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+    When logout of Manager
+    
+    Then add a Processor to the Bridge "mybridge" with body is failing with HTTP response code 401:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "parameters": {
+            "topic":  "myKafkaTopic"
+        },
+        "type": "KafkaTopic"
+      }
+    }
+    """
+
+  Scenario: Cannot access processor details without authentication
+    Given authenticate against Manager
+    And create a new Bridge "mybridge"
+    And the Bridge "mybridge" is existing with status "ready" within 2 minutes
+    And the Ingress of Bridge "mybridge" is available within 2 minutes
+
+    And add a Processor to the Bridge "mybridge" with body:
+    """
+    {
+      "name": "myProcessor",
+      "action": {
+        "parameters": {
+            "topic":  "myKafkaTopic"
+        },
+        "type": "KafkaTopic"
+      }
+    }
+    """
+    And the Processor "myProcessor" of the Bridge "mybridge" is existing with status "ready" within 3 minutes
+
+    When logout of Manager
+
+    Then get Processor "myProcessor" of the Bridge "mybridge" is failing with HTTP response code 401
```

Comment:
Can you please add a scenario with deleting non existing processor? It should be fixed by https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/396

Reply:
I get a 500 ...

Reply:
logs: 
```
Request method:	DELETE
Request URI:	http://192.168.49.2:80/manager/api/v1/bridges/ff79166a-cf6f-4269-9a9e-2f8a2c6c01f6/processors/myProcessor-3239
Proxy:			<none>
Request params:	<none>
Query params:	<none>
Form params:	<none>
Path params:	<none>
Headers:		Accept=*/*
				Content-Type=application/json
Cookies:		<none>
Multiparts:		<none>
Body:			<none>

HTTP/1.1 500 Internal Server Error
Date: Fri, 04 Mar 2022 14:52:56 GMT
Content-Type: application/json; charset=utf-8
Content-Length: 72
Connection: keep-alive

{
    "details": "Error id 9853e7ea-b1eb-4aa8-ae86-af62a1f3684c-3",
    "stack": ""
}
```
```
2022-03-04 14:52:56,404 ERROR [io.qua.ver.htt.run.QuarkusErrorHandler] (executor-thread-11) HTTP Request to /api/v1/bridges/ff79166a-cf6f-4269-9a9e-2f8a2c6c01f6/processors/myProcessor-3239 failed, error id: 9853e7ea-b1eb-4aa8-ae86-af62a1f3684c-3: org.jboss.resteasy.spi.UnhandledException: java.lang.NullPointerException
	at org.jboss.resteasy.core.ExceptionHandler.handleApplicationException(ExceptionHandler.java:105)
	at org.jboss.resteasy.core.ExceptionHandler.handleException(ExceptionHandler.java:359)
	at org.jboss.resteasy.core.SynchronousDispatcher.writeException(SynchronousDispatcher.java:218)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:519)
	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$invoke$4(SynchronousDispatcher.java:261)
	at org.jboss.resteasy.core.SynchronousDispatcher.lambda$preprocess$0(SynchronousDispatcher.java:161)
	at org.jboss.resteasy.core.interception.jaxrs.PreMatchContainerRequestContext.filter(PreMatchContainerRequestContext.java:364)
	at org.jboss.resteasy.core.SynchronousDispatcher.preprocess(SynchronousDispatcher.java:164)
	at org.jboss.resteasy.core.SynchronousDispatcher.invoke(SynchronousDispatcher.java:247)
	at io.quarkus.resteasy.runtime.standalone.RequestDispatcher.service(RequestDispatcher.java:73)
	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler.dispatch(VertxRequestHandler.java:151)
	at io.quarkus.resteasy.runtime.standalone.VertxRequestHandler$1.run(VertxRequestHandler.java:91)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$13.runWith(VertxCoreRecorder.java:543)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2449)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1478)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.NullPointerException
	at com.redhat.service.bridge.manager.ProcessorServiceImpl.updateProcessorConnectorForDeletionOnDB(ProcessorServiceImpl.java:189)
	at com.redhat.service.bridge.manager.ProcessorServiceImpl_Subclass.updateProcessorConnectorForDeletionOnDB$$superforward1(Unknown Source)
	at com.redhat.service.bridge.manager.ProcessorServiceImpl_Subclass$$function$$5.apply(Unknown Source)
	at io.quarkus.arc.impl.AroundInvokeInvocationContext.proceed(AroundInvokeInvocationContext.java:54)

```

Reply:
or maybe I don't have the latest version. gonna check that

Reply:
probably

Reply:
verified OK with latest version :)